### PR TITLE
Vet Clinic Database

### PR DESCRIPTION
**Pull request by GutemaG**
## Vet clinic database:
- Create vet_clinic database
- Create animals table in vet_clinic database
- Insert data into animals table
- Retrieve data from animal table.

Here are some Screenshots from the queries:
query 1
query 2-
query 3-
query 4
query 5-
query 8
query 9
query 10

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+	
+
+INSERT INTO animals(
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	values( 3, 'Pikachu', 'Jan 7, 2021', 1, false, 15.04);
+
+INSERT INTO animals(
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	values( 4, 'Devimon', 'May 12, 2017', 5, true, 15);
+
```
Comment (mahbubul14) [inline data.sql]: - [x] Devimon weighs 11kg. kindly change the value. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @GutemaG,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: Commit messages are good.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by GutemaG** [comment]
### Some Query screen shot after change the RV
new
new -2

**Review by pinkmoon25 — APPROVED** [review]
Hi @GutemaG , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
data, schema and query SQL :heavy_check_mark: 
github flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**